Makefile

```diff
@@ -55,14 +58,18 @@ bundle: ;
 .PHONY: bundle
 
 fetch-junctions:
-	$(BST) $(BST_ARGS) $(_BST_ARGS) fetch freedesktop-sdk.bst gnome-sdk.bst
+	$(BST) $(BST_ARGS) $(_BST_ARGS) fetch $(JUNCTION_BST_FILES)
 .PHONY: fetch-junctions
 
 track:
-	$(BST) $(BST_ARGS) $(_BST_ARGS) track freedesktop-sdk.bst gnome-sdk.bst
-	$(BST) $(BST_ARGS) $(_BST_ARGS) track flatpak-runtimes.bst flatpak-platform-extensions.bst --deps=all
+	$(BST) $(BST_ARGS) $(_BST_ARGS) track $(JUNCTION_BST_FILES)
+	$(BST) $(BST_ARGS) $(_BST_ARGS) track $(TOPLEVEL_BST_FILES) --deps=all
 .PHONY: track
 
+push:
+	$(BST) $(BST_ARGS) $(_BST_ARGS) push $(TOPLEVEL_BST_FILES) --deps=all
```

Comment:
Might need to bump the 100GB of storage with `--deps=all`, but we can give it a shot.